blivet/tasks/fsminsize.py

```diff
@@ -54,7 +54,7 @@ def _getResizeInfo(self):
         except OSError as e:
             error_msg = "failed to gather info from resize program: %s" % e
         if rc:
-            error_msg = "failed to gather info from resize program: %s" % e
+            error_msg = "failed to gather info from resize program: %d" % rc
```

Comment:
Seems like this would be better inside the try: just so it's clear it only runs when it succeeds.